Java/src/com/company/Objects.java

```diff
@@ -0,0 +1,45 @@
+package com.company;
+
+public class Objects {
+    //our prewritten fields
+    private String name;
+    private String city;
+    private int born;
+
+    public Objects() {}

```

Comment:
IMHO you should rename `Objects` to some more appropriate class name, `Person`, for example.